### ISSUE TITLE
Send email when review has been added to queue

### ISSUE THREAD

**Issue by Hadrien-lcrx** [udacity/reviewer-dashboard-feedback#33]

**Comment by Hadrien-lcrx** [comment]
When entering the queue for reviews, if reviews are added, reviewers are not notified. As a new reviewer, I entered the queue, and not receiving any email, didn't go back on the dashboard until the next evening to join the queue again. I then realized I had three reviews expired! Additionally, I didn't know what to do about these issues since I could not click on them (see issue 34).

**Comment by kartikarora** [comment]
Email is too much. Desktop #19 and/or mobile notifications #9 are better

**Comment by kartikarora** [comment]
Also [urcli](https://github.com/trolster/urcli) added `notify` command which will work till the time API is working

**Comment by romcabrera** [comment]
Right now I receive email notifications for Mentor messages, I think it makes sense to send emails for assigned reviews also (of course you could opt out if desired)

**Comment by Hadrien-lcrx** [comment]
@kartikarora I share @romcabrera's opinion, I receive an email notification when my mentees message me, so wouldn't feel overwhelmed with email notifications for assigned reviews, and an option to opt out might solve the issue. I'll add that I find Desktop notifications more invasive than email (I only allows Slack messages notifications on my computer, and SMS / Phone notifications on my mobile).

**Comment by Hadrien-lcrx** [comment]
Alternatively, for projects where it's verified that once queuing, at least 5 projects get assigned under 12 hours, you could decide to just send an email when the queue is full.

**Comment by taloc** [comment]
@Hadrien-Icrx I totally agree that an email notification is required when a project is assigned for review. In my specific case I have day job and I'm not always in my desktop checking the Reviewer Dashboard. An email notification will allow me to take action quicker than today.
